grizli/prep.py

```diff
@@ -145,6 +155,30 @@ def fresh_flt_file(file, preserve_dq=False, path='../RAW/', verbose=True, extra_
                                calib_types=calib_types,
                                verbose=False)
 
+    if filter in ['GR150C', 'GR150R']: 
```

Comment:
Getting these directly from CDBS can wait, but this could be cleaned up into a dictionary for now,  like 
```python
niriss_flats = {'GR150R': 
                         {'F115W': 'jwst_niriss_flat_0202.fits', 
                          'F150W': 'jwst_niriss_flat_0187.fits', 
                          ...}, 
                'GR150C': ...
                }

flat_file = niriss_flats[FILTER][PUPIL]
```

Note that `jref` is usually for HST ACS (`jref` for WFC3), but OK to use as shorthand for now until it's fully implemented with CRDS.

Or another cleaner way, here specified as a YAML string but that could also be in a file like `grizli/data/jwst_references.yml`.  But then this starts to look like what crds is doing anyway....

```python
import yaml # at the top

_JWST_FLATS = """
NIRISS:
  GR150R:
    F115W: jwst_niriss_flat_0202.fits
    F150W: jwst_niriss_flat_0187.fits
  GR150C:
    F115W: xxx.fits
NIRCAM:
  F115W:
      GRISMR: yyy.fits
      GRISMC: zzz.fits
"""
jwst_flats = yaml.safe_load(_JWST_FLATS)
flat_file = jwst_flats[INSTRUMENT][FILTER][PUPIL]
```